[2023-09-23T21:56:50.359-0400] {processor.py:157} INFO - Started process (PID=1225) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T21:56:50.360-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T21:56:50.362-0400] {logging_mixin.py:151} INFO - [2023-09-23T21:56:50.361-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T21:56:50.367-0400] {logging_mixin.py:151} INFO - [2023-09-23T21:56:50.366-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T21:56:50.368-0400] {logging_mixin.py:151} INFO - [2023-09-23T21:56:50.367-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T21:56:50.368-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T21:56:50.383-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-09-23T21:57:39.586-0400] {processor.py:157} INFO - Started process (PID=1784) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T21:57:39.587-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T21:57:39.588-0400] {logging_mixin.py:151} INFO - [2023-09-23T21:57:39.588-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T21:57:39.591-0400] {logging_mixin.py:151} INFO - [2023-09-23T21:57:39.590-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T21:57:39.591-0400] {logging_mixin.py:151} INFO - [2023-09-23T21:57:39.591-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T21:57:39.592-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T21:57:39.600-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2023-09-23T21:58:27.267-0400] {processor.py:157} INFO - Started process (PID=2029) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T21:58:27.270-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T21:58:27.271-0400] {logging_mixin.py:151} INFO - [2023-09-23T21:58:27.271-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T21:58:27.278-0400] {logging_mixin.py:151} INFO - [2023-09-23T21:58:27.276-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T21:58:27.278-0400] {logging_mixin.py:151} INFO - [2023-09-23T21:58:27.278-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T21:58:27.279-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T21:58:27.297-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2023-09-23T21:59:22.454-0400] {processor.py:157} INFO - Started process (PID=2307) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T21:59:22.456-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T21:59:22.458-0400] {logging_mixin.py:151} INFO - [2023-09-23T21:59:22.457-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T21:59:22.465-0400] {logging_mixin.py:151} INFO - [2023-09-23T21:59:22.464-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T21:59:22.466-0400] {logging_mixin.py:151} INFO - [2023-09-23T21:59:22.465-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T21:59:22.466-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T21:59:22.486-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2023-09-23T22:00:15.781-0400] {processor.py:157} INFO - Started process (PID=2985) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:00:15.783-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:00:15.784-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:00:15.784-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:00:15.787-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:00:15.786-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:00:15.787-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:00:15.787-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:00:15.788-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:00:15.799-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-09-23T22:01:10.014-0400] {processor.py:157} INFO - Started process (PID=3333) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:01:10.020-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:01:10.021-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:01:10.020-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:01:10.026-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:01:10.025-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:01:10.026-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:01:10.026-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:01:10.027-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:01:10.040-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-09-23T22:02:01.616-0400] {processor.py:157} INFO - Started process (PID=3651) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:02:01.619-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:02:01.620-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:02:01.619-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:02:01.625-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:02:01.624-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:02:01.626-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:02:01.625-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:02:01.626-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:02:01.636-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-09-23T22:02:53.143-0400] {processor.py:157} INFO - Started process (PID=3910) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:02:53.145-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:02:53.146-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:02:53.146-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:02:53.151-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:02:53.150-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:02:53.151-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:02:53.151-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:02:53.152-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:02:53.165-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-09-23T22:03:49.009-0400] {processor.py:157} INFO - Started process (PID=4183) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:03:49.014-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:03:49.016-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:03:49.015-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:03:49.022-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:03:49.021-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:03:49.023-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:03:49.022-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:03:49.023-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:03:49.038-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2023-09-23T22:04:44.582-0400] {processor.py:157} INFO - Started process (PID=4454) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:04:44.585-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:04:44.586-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:04:44.586-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:04:44.590-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:04:44.590-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:04:44.591-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:04:44.590-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:04:44.591-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:04:44.604-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-09-23T22:05:36.757-0400] {processor.py:157} INFO - Started process (PID=4772) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:05:36.761-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:05:36.762-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:05:36.761-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:05:36.769-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:05:36.768-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:05:36.770-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:05:36.770-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:05:36.771-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:05:36.787-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-09-23T22:06:29.864-0400] {processor.py:157} INFO - Started process (PID=5158) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:06:29.866-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:06:29.868-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:06:29.867-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:06:29.874-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:06:29.873-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:06:29.875-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:06:29.875-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:06:29.875-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:06:29.884-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-09-23T22:07:19.056-0400] {processor.py:157} INFO - Started process (PID=5370) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:07:19.059-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:07:19.060-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:07:19.059-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:07:19.063-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:07:19.063-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:07:19.064-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:07:19.063-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:07:19.064-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:07:19.072-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2023-09-23T22:08:10.870-0400] {processor.py:157} INFO - Started process (PID=5602) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:08:10.873-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:08:10.874-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:08:10.874-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:08:10.877-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:08:10.877-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:08:10.877-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:08:10.877-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:08:10.878-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:08:10.885-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2023-09-23T22:09:05.444-0400] {processor.py:157} INFO - Started process (PID=5871) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:09:05.447-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:09:05.449-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:09:05.448-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:09:05.453-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:09:05.452-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:09:05.453-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:09:05.453-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:09:05.453-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:09:05.463-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-09-23T22:09:59.149-0400] {processor.py:157} INFO - Started process (PID=6135) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:09:59.153-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:09:59.154-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:09:59.154-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:09:59.157-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:09:59.157-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:09:59.158-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:09:59.158-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:09:59.158-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:09:59.167-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-09-23T22:10:50.769-0400] {processor.py:157} INFO - Started process (PID=6654) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:10:50.773-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:10:50.774-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:10:50.773-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:10:50.779-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:10:50.779-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:10:50.780-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:10:50.779-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:10:50.780-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:10:50.788-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-09-23T22:11:43.023-0400] {processor.py:157} INFO - Started process (PID=7030) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:11:43.025-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:11:43.027-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:11:43.026-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:11:43.032-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:11:43.031-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:11:43.033-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:11:43.033-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:11:43.033-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:11:43.048-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-09-23T22:12:34.989-0400] {processor.py:157} INFO - Started process (PID=7292) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:12:34.993-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:12:34.994-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:12:34.993-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:12:34.999-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:12:34.998-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:12:34.999-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:12:34.999-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:12:34.999-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:12:35.013-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-09-23T22:13:28.840-0400] {processor.py:157} INFO - Started process (PID=7555) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:13:28.845-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:13:28.846-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:13:28.846-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:13:28.850-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:13:28.849-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:13:28.850-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:13:28.850-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:13:28.850-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:13:28.858-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-09-23T22:14:23.073-0400] {processor.py:157} INFO - Started process (PID=7819) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:14:23.078-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:14:23.080-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:14:23.079-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:14:23.085-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:14:23.084-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:14:23.086-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:14:23.086-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:14:23.086-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:14:23.098-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-09-23T22:15:14.599-0400] {processor.py:157} INFO - Started process (PID=8018) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:15:14.603-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:15:14.604-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:15:14.604-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:15:14.608-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:15:14.607-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:15:14.608-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:15:14.608-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:15:14.609-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:15:14.621-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-09-23T22:16:09.024-0400] {processor.py:157} INFO - Started process (PID=8284) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:16:09.028-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:16:09.030-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:16:09.029-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:16:09.038-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:16:09.037-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:16:09.038-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:16:09.038-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:16:09.039-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:16:09.051-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-09-23T22:17:04.086-0400] {processor.py:157} INFO - Started process (PID=8612) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:17:04.090-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:17:04.091-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:17:04.091-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:17:04.095-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:17:04.095-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:17:04.096-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:17:04.096-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:17:04.096-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:17:04.109-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-09-23T22:17:56.184-0400] {processor.py:157} INFO - Started process (PID=8873) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:17:56.187-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:17:56.189-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:17:56.188-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:17:56.194-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:17:56.193-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:17:56.194-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:17:56.194-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:17:56.194-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:17:56.206-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-09-23T22:18:48.604-0400] {processor.py:157} INFO - Started process (PID=9130) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:18:48.608-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:18:48.609-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:18:48.608-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:18:48.614-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:18:48.613-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:18:48.614-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:18:48.614-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:18:48.614-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:18:48.625-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-09-23T22:19:44.535-0400] {processor.py:157} INFO - Started process (PID=9395) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:19:44.538-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:19:44.539-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:19:44.539-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:19:44.547-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:19:44.545-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:19:44.547-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:19:44.547-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:19:44.548-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:19:44.564-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-09-23T22:20:32.016-0400] {processor.py:157} INFO - Started process (PID=9637) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:20:32.020-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:20:32.021-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:20:32.020-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:20:32.023-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:20:32.023-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:20:32.024-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:20:32.023-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:20:32.024-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:20:32.035-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-09-23T22:21:25.092-0400] {processor.py:157} INFO - Started process (PID=9925) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:21:25.096-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:21:25.097-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:21:25.097-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:21:25.102-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:21:25.102-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:21:25.103-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:21:25.103-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:21:25.103-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:21:25.118-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-09-23T22:22:18.106-0400] {processor.py:157} INFO - Started process (PID=10157) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:22:18.110-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:22:18.112-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:22:18.111-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:22:18.116-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:22:18.115-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:22:18.116-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:22:18.116-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:22:18.116-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:22:18.128-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-09-23T22:23:09.785-0400] {processor.py:157} INFO - Started process (PID=10413) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:23:09.789-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:23:09.791-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:23:09.790-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:23:09.795-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:23:09.795-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:23:09.796-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:23:09.795-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:23:09.796-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:23:09.806-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-09-23T22:24:00.438-0400] {processor.py:157} INFO - Started process (PID=10661) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:24:00.444-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:24:00.446-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:24:00.445-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:24:00.455-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:24:00.454-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:24:00.456-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:24:00.456-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:24:00.457-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:24:00.472-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.042 seconds
[2023-09-23T22:24:50.568-0400] {processor.py:157} INFO - Started process (PID=10918) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:24:50.572-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:24:50.573-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:24:50.572-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:24:50.577-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:24:50.576-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:24:50.577-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:24:50.577-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:24:50.577-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:24:50.586-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-09-23T22:25:42.753-0400] {processor.py:157} INFO - Started process (PID=11175) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:25:42.757-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:25:42.758-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:25:42.758-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:25:42.762-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:25:42.761-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:25:42.762-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:25:42.762-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:25:42.762-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:25:42.773-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-09-23T22:26:31.600-0400] {processor.py:157} INFO - Started process (PID=11477) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:26:31.604-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:26:31.605-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:26:31.605-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:26:31.608-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:26:31.608-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:26:31.608-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:26:31.608-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:26:31.609-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:26:31.617-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2023-09-23T22:27:18.939-0400] {processor.py:157} INFO - Started process (PID=11658) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:27:18.944-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:27:18.946-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:27:18.945-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:27:18.951-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:27:18.950-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:27:18.951-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:27:18.951-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:27:18.952-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:27:18.962-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-09-23T22:28:09.668-0400] {processor.py:157} INFO - Started process (PID=11915) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:28:09.673-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:28:09.674-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:28:09.674-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:28:09.681-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:28:09.679-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:28:09.681-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:28:09.681-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:28:09.682-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:28:09.700-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2023-09-23T22:29:03.217-0400] {processor.py:157} INFO - Started process (PID=12172) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:29:03.223-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:29:03.225-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:29:03.224-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:29:03.232-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:29:03.231-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:29:03.232-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:29:03.232-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:29:03.233-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:29:03.245-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2023-09-23T22:29:56.607-0400] {processor.py:157} INFO - Started process (PID=12437) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:29:56.611-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:29:56.613-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:29:56.613-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:29:56.621-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:29:56.619-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:29:56.621-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:29:56.621-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:29:56.622-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:29:56.635-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-09-23T22:30:44.490-0400] {processor.py:157} INFO - Started process (PID=12679) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:30:44.494-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:30:44.496-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:30:44.495-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:30:44.502-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:30:44.501-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:30:44.503-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:30:44.502-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:30:44.503-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:30:44.525-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2023-09-23T22:31:36.357-0400] {processor.py:157} INFO - Started process (PID=13119) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:31:36.360-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:31:36.362-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:31:36.361-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:31:36.366-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:31:36.365-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:31:36.366-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:31:36.366-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:31:36.366-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:31:36.380-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-09-23T22:32:27.630-0400] {processor.py:157} INFO - Started process (PID=13376) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:32:27.634-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:32:27.636-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:32:27.635-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:32:27.640-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:32:27.639-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:32:27.641-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:32:27.641-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:32:27.641-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:32:27.655-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-09-23T22:33:14.713-0400] {processor.py:157} INFO - Started process (PID=13557) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:33:14.717-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:33:14.718-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:33:14.718-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:33:14.721-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:33:14.721-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:33:14.721-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:33:14.721-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:33:14.722-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:33:14.730-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2023-09-23T22:34:02.529-0400] {processor.py:157} INFO - Started process (PID=13798) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:34:02.533-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:34:02.536-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:34:02.535-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:34:02.543-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:34:02.542-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:34:02.543-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:34:02.543-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:34:02.544-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:34:02.555-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-09-23T22:34:53.390-0400] {processor.py:157} INFO - Started process (PID=14279) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:34:53.392-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:34:53.393-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:34:53.393-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:34:53.396-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:34:53.395-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:34:53.397-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:34:53.396-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:34:53.397-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:34:53.409-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-09-23T22:35:47.458-0400] {processor.py:157} INFO - Started process (PID=14603) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:35:47.463-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:35:47.464-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:35:47.464-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:35:47.472-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:35:47.471-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:35:47.472-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:35:47.472-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:35:47.473-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:35:47.487-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-09-23T22:36:34.594-0400] {processor.py:157} INFO - Started process (PID=15002) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:36:34.598-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:36:34.600-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:36:34.600-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:36:34.606-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:36:34.605-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:36:34.607-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:36:34.606-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:36:34.607-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:36:34.623-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-09-23T22:37:26.589-0400] {processor.py:157} INFO - Started process (PID=15258) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:37:26.593-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:37:26.594-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:37:26.593-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:37:26.596-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:37:26.596-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:37:26.597-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:37:26.596-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:37:26.597-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:37:26.606-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2023-09-23T22:38:15.567-0400] {processor.py:157} INFO - Started process (PID=15440) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:38:15.571-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:38:15.573-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:38:15.572-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:38:15.577-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:38:15.577-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:38:15.578-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:38:15.578-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:38:15.578-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:38:15.593-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-09-23T22:39:10.601-0400] {processor.py:157} INFO - Started process (PID=15712) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:39:10.604-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:39:10.605-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:39:10.605-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:39:10.608-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:39:10.608-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:39:10.608-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:39:10.608-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:39:10.608-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:39:10.624-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-09-23T22:40:02.446-0400] {processor.py:157} INFO - Started process (PID=15963) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:40:02.450-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:40:02.451-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:40:02.451-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:40:02.455-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:40:02.455-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:40:02.456-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:40:02.456-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:40:02.456-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:40:02.466-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-09-23T22:40:53.547-0400] {processor.py:157} INFO - Started process (PID=16219) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:40:53.551-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:40:53.553-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:40:53.552-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:40:53.560-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:40:53.559-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:40:53.561-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:40:53.561-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:40:53.562-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:40:53.576-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-09-23T22:41:48.666-0400] {processor.py:157} INFO - Started process (PID=16624) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:41:48.670-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:41:48.671-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:41:48.671-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:41:48.674-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:41:48.673-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:41:48.674-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:41:48.674-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:41:48.674-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:41:48.687-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-09-23T22:42:44.413-0400] {processor.py:157} INFO - Started process (PID=16895) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:42:44.416-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:42:44.417-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:42:44.417-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:42:44.420-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:42:44.419-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:42:44.420-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:42:44.420-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:42:44.421-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:42:44.431-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-09-23T22:43:42.143-0400] {processor.py:157} INFO - Started process (PID=17171) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:43:42.146-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:43:42.147-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:43:42.147-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:43:42.150-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:43:42.150-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:43:42.151-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:43:42.151-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:43:42.151-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:43:42.160-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2023-09-23T22:44:36.736-0400] {processor.py:157} INFO - Started process (PID=17440) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:44:36.741-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:44:36.742-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:44:36.742-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:44:36.746-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:44:36.746-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:44:36.747-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:44:36.747-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:44:36.747-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:44:36.759-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-09-23T22:45:27.298-0400] {processor.py:157} INFO - Started process (PID=17690) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:45:27.302-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:45:27.303-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:45:27.303-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:45:27.307-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:45:27.307-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:45:27.308-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:45:27.308-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:45:27.308-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:45:27.321-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-09-23T22:46:21.540-0400] {processor.py:157} INFO - Started process (PID=17919) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:46:21.543-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:46:21.544-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:46:21.543-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:46:21.548-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:46:21.547-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:46:21.548-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:46:21.548-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:46:21.548-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:46:21.559-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-09-23T22:47:16.825-0400] {processor.py:157} INFO - Started process (PID=18244) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:47:16.829-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:47:16.831-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:47:16.831-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:47:16.837-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:47:16.836-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:47:16.837-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:47:16.837-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:47:16.837-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:47:16.853-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-09-23T22:48:09.717-0400] {processor.py:157} INFO - Started process (PID=18550) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:48:09.721-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:48:09.722-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:48:09.721-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:48:09.725-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:48:09.724-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:48:09.725-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:48:09.725-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:48:09.725-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:48:09.737-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-09-23T22:48:59.525-0400] {processor.py:157} INFO - Started process (PID=18976) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:48:59.529-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:48:59.531-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:48:59.530-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:48:59.535-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:48:59.534-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:48:59.536-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:48:59.536-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:48:59.536-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:48:59.547-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-09-23T22:49:53.704-0400] {processor.py:157} INFO - Started process (PID=19331) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:49:53.706-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:49:53.708-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:49:53.707-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:49:53.714-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:49:53.713-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:49:53.714-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:49:53.714-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:49:53.715-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:49:53.726-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-09-23T22:50:48.775-0400] {processor.py:157} INFO - Started process (PID=19746) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:50:48.780-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:50:48.781-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:50:48.781-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:50:48.785-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:50:48.785-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:50:48.786-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:50:48.786-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:50:48.786-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:50:48.799-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-09-23T22:51:44.506-0400] {processor.py:157} INFO - Started process (PID=20115) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:51:44.512-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:51:44.513-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:51:44.513-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:51:44.517-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:51:44.516-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-09-23T22:51:44.518-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:51:44.517-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-09-23T22:51:44.518-0400] {processor.py:841} WARNING - No viable dags retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:51:44.530-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-09-23T22:52:39.184-0400] {processor.py:157} INFO - Started process (PID=20557) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:52:39.186-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:52:39.189-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:52:39.188-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:52:39.472-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:52:39.571-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:52:39.571-0400] {manager.py:499} INFO - Created Permission View: %s
[2023-09-23T22:52:39.585-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:52:39.585-0400] {manager.py:499} INFO - Created Permission View: %s
[2023-09-23T22:52:39.597-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:52:39.596-0400] {manager.py:499} INFO - Created Permission View: %s
[2023-09-23T22:52:39.597-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:52:39.597-0400] {security.py:708} INFO - Not syncing DAG-level permissions for DAG 'DAG:example_local_kubernetes_executor' as access control is unset.
[2023-09-23T22:52:39.598-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:52:39.597-0400] {dag.py:2915} INFO - Sync 1 DAGs
[2023-09-23T22:52:39.607-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:52:39.607-0400] {dag.py:2937} INFO - Creating ORM DAG for example_local_kubernetes_executor
[2023-09-23T22:52:39.607-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:52:39.607-0400] {dag.py:3696} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-09-23T22:52:39.629-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.451 seconds
[2023-09-23T22:53:41.262-0400] {processor.py:157} INFO - Started process (PID=20946) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:53:41.266-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:53:41.267-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:53:41.267-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:53:41.499-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:53:41.515-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:53:41.514-0400] {dag.py:2915} INFO - Sync 1 DAGs
[2023-09-23T22:53:41.522-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:53:41.522-0400] {dag.py:3696} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-09-23T22:53:41.544-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.287 seconds
[2023-09-23T22:54:34.657-0400] {processor.py:157} INFO - Started process (PID=21252) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:54:34.662-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:54:34.663-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:54:34.663-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:54:34.944-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:54:34.967-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:54:34.966-0400] {dag.py:2915} INFO - Sync 1 DAGs
[2023-09-23T22:54:34.976-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:54:34.976-0400] {dag.py:3696} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-09-23T22:54:35.002-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.352 seconds
[2023-09-23T22:55:34.029-0400] {processor.py:157} INFO - Started process (PID=21586) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:55:34.033-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:55:34.035-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:55:34.034-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:55:34.263-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:55:34.279-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:55:34.279-0400] {dag.py:2915} INFO - Sync 1 DAGs
[2023-09-23T22:55:34.287-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:55:34.287-0400] {dag.py:3696} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-09-23T22:55:34.310-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.285 seconds
[2023-09-23T22:56:30.370-0400] {processor.py:157} INFO - Started process (PID=21968) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:56:30.374-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:56:30.375-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:56:30.375-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:56:30.605-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:56:30.621-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:56:30.620-0400] {dag.py:2915} INFO - Sync 1 DAGs
[2023-09-23T22:56:30.628-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:56:30.628-0400] {dag.py:3696} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-09-23T22:56:30.651-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.289 seconds
[2023-09-23T22:57:27.372-0400] {processor.py:157} INFO - Started process (PID=22288) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:57:27.376-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:57:27.378-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:57:27.377-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:57:27.619-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:57:27.638-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:57:27.637-0400] {dag.py:2915} INFO - Sync 1 DAGs
[2023-09-23T22:57:27.645-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:57:27.645-0400] {dag.py:3696} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-09-23T22:57:27.668-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.303 seconds
[2023-09-23T22:58:25.829-0400] {processor.py:157} INFO - Started process (PID=22615) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:58:25.833-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:58:25.834-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:58:25.834-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:58:26.096-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:58:26.114-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:58:26.113-0400] {dag.py:2915} INFO - Sync 1 DAGs
[2023-09-23T22:58:26.122-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:58:26.122-0400] {dag.py:3696} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-09-23T22:58:26.144-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.317 seconds
[2023-09-23T22:59:18.166-0400] {processor.py:157} INFO - Started process (PID=22860) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:59:18.172-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T22:59:18.174-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:59:18.173-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:59:18.412-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T22:59:18.428-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:59:18.428-0400] {dag.py:2915} INFO - Sync 1 DAGs
[2023-09-23T22:59:18.435-0400] {logging_mixin.py:151} INFO - [2023-09-23T22:59:18.435-0400] {dag.py:3696} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-09-23T22:59:18.460-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.301 seconds
[2023-09-23T23:00:18.627-0400] {processor.py:157} INFO - Started process (PID=23189) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:00:18.631-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T23:00:18.633-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:00:18.632-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:00:18.876-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:00:18.895-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:00:18.894-0400] {dag.py:2915} INFO - Sync 1 DAGs
[2023-09-23T23:00:18.902-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:00:18.901-0400] {dag.py:3696} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-09-23T23:00:18.927-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.306 seconds
[2023-09-23T23:01:18.480-0400] {processor.py:157} INFO - Started process (PID=23573) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:01:18.485-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T23:01:18.486-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:01:18.486-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:01:18.740-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:01:18.759-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:01:18.759-0400] {dag.py:2915} INFO - Sync 1 DAGs
[2023-09-23T23:01:18.767-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:01:18.767-0400] {dag.py:3696} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-09-23T23:01:18.791-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.317 seconds
[2023-09-23T23:02:14.254-0400] {processor.py:157} INFO - Started process (PID=23906) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:02:14.258-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T23:02:14.259-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:02:14.258-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:02:14.507-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:02:14.526-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:02:14.525-0400] {dag.py:2915} INFO - Sync 1 DAGs
[2023-09-23T23:02:14.535-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:02:14.535-0400] {dag.py:3696} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-09-23T23:02:14.563-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.311 seconds
[2023-09-23T23:03:12.888-0400] {processor.py:157} INFO - Started process (PID=24179) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:03:12.891-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T23:03:12.892-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:03:12.892-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:03:13.184-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:03:13.204-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:03:13.204-0400] {dag.py:2915} INFO - Sync 1 DAGs
[2023-09-23T23:03:13.214-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:03:13.214-0400] {dag.py:3696} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-09-23T23:03:13.237-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.353 seconds
[2023-09-23T23:04:09.552-0400] {processor.py:157} INFO - Started process (PID=24452) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:04:09.557-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T23:04:09.560-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:04:09.559-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:04:09.808-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:04:09.826-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:04:09.825-0400] {dag.py:2915} INFO - Sync 1 DAGs
[2023-09-23T23:04:09.833-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:04:09.833-0400] {dag.py:3696} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-09-23T23:04:09.856-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.313 seconds
[2023-09-23T23:05:04.779-0400] {processor.py:157} INFO - Started process (PID=24911) to work on /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:05:04.782-0400] {processor.py:829} INFO - Processing file /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-09-23T23:05:04.784-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:05:04.783-0400] {dagbag.py:539} INFO - Filling up the DagBag from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:05:05.067-0400] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-09-23T23:05:05.090-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:05:05.089-0400] {dag.py:2915} INFO - Sync 1 DAGs
[2023-09-23T23:05:05.098-0400] {logging_mixin.py:151} INFO - [2023-09-23T23:05:05.098-0400] {dag.py:3696} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-09-23T23:05:05.121-0400] {processor.py:179} INFO - Processing /home/beau/airflow/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.348 seconds
